streaming/README.md

```diff
@@ -511,4 +511,100 @@ metrics, following the same pattern of the receiving side.
+The slave Netdata will log a majority of error messages.
+
+```
+netdata ERROR : STREAM_SENDER[SLAVE HOSTNAME] : STREAM SLAVE HOSTNAME [send to MASTER IP:MASTER PORT]: too many data pending - buffer is X bytes long,
+Y unsent - we have sent Z bytes in total, W on this connection. Closing connection to flush the data.
+```
+
+The the value X, Y, Z and W are integer numbers.
+
+On the other hand, in the master side you can have different reports where the most common message that you can see is
+
+```
+netdata ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : read failed: end of file
+```
+
+Another common problem is the slave sends only part of the message for the master. In this case, the master
+will not have condition to understand the message received and it will write the following inside `error.log`:
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : sent command 'B' which is not known by netdata, for host 'HOSTNAME'. Disabling it.
+```
+
+In this example, 'B' was part of a 'BEGIN' message that was cut due connection problems.
+
+The last problem related to slow connections happens when some message was missed and the following commands received had a
+direct connection with it. For example, when the slave could not send the list of charts available on its host and it
+sends in the sequence a 'SET' message on a chart that was missed, the master will show a messages like this:
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : requested a SET on chart 'CHART NAME' of host 'HOSTNAME', without a dimension. Disabling it.
+```
+
+### Slave cannot connect to master
+
+Sometimes you are type writing the address and port of the master inside stream.conf and you write a wrong information,
+Netdata will write a message like this in the slave log
+
+```
+ERROR : STREAM_SENDER[HOSTNAME] : Failed to connect to 'MASTER IP', port 'MASTER PORT' (errno 113, No route to host)
+```
+
+This error can also happen case the firewall on master is not allowing connections to the master port.
+
+### Is this a Netdata?
+
+This question can appear when Netdata starts the stream and receives an unexpected response when connecting to the slave for the first time. 
+This error can appear for a number of different reasons, but the most common is that the master node has SSL activated and the slave sent a plain text response. 
+The complete error message will look like this:
+
+```
+ERROR : STREAM_SENDER[SLAVE HOSTNAME] : STREAM SLAVE HOSTNAME [send to MASTER HOSTNAME:MASTER PORT]: server is not replying properly (is it a netdata?).
+```
+
+### Stream charts wrong
+
+When the slave sends data to master, the charts need to be aligned during the same interval of time. If there are gaps in metrics collection,
+these gaps need to be consistent between slave and master nodes. If there are differences between charts, it often means that your master's `memory mode` is not configured correctly.
```

Comment:
I have no idea what you're saying here regarding the gaps and what the solution is. What do you mean not configured correctly? The whole thing needs to be stated differently.

Reply:
I rewrite this and I also set a link to the documentation for the user see where he can find the correct adjusts for memory mode.